googleapiclient/discovery_cache/__init__.py

```diff
@@ -47,3 +48,36 @@ def autodetect():
     except Exception as e:
         LOGGER.warning(e, exc_info=True)
         return None
+
+def get_static_doc(uri):
+    """Retrieves the discovery document from the directory defined in
+    DISCOVERY_DOC_STATIC_DIR corresponding to the uri provided.
```

Comment:
Variable above is named `DISCOVERY_DOC_DIR`

Reply:
Good catch! Fixed.